class.dable.php

```diff
@@ -140,8 +140,12 @@ public function get_thumbnail( $post ) {
 			// Does it has dable-og-thumbnail size image?
 			$attach_data = wp_get_attachment_metadata( $image_id, true );
 
+			$og_options = get_option( 'dable-og-settings', array() );
+			$org_thumbnail_size = $og_options['thumbnail_size'];
+			$curr_thumbnail_size = $attach_data['sizes']['dable-og-thumbnail']['width'];
```

Comment:
기존 코드를 보면,

```
if ( ! is_array($attach_data) || ! isset($attach_data['sizes']['dable-og-thumbnail']) ) {
```

`$attach_data` 가 array가 아닌 경우도 있나봐요 (아마도 null 같은?)
그래서 여기서도 null인 경우에 대한 처리가 필요할 것 같아요.

```suggestion
			$curr_thumbnail_size = is_array($attach_data) ? $attach_data['sizes']['dable-og-thumbnail']['width'] : null;
```

Reply:
아마 아실 수도 있겠지만 php의 array는 JS의 array와 object를 합쳐놓은 만능 혼종입니다. 😂
https://www.php.net/manual/en/language.types.array.php

Reply:
리뷰 감사합니다 @goonookim 님 🙌 
php array는,, 처음 알았습니다 🤯 

리뷰주신 부분 반영했고 모호했던 변수명 보완하면서
섬네일 세팅 변경여부를 확인하는 로직에서 놓친 내용을 아래와 같이 수정했습니다. 추가로 확인 부탁드립니다! 🙇 

기존PR)
- 기존 섬네일 사이즈를 width 기준으로만 확인함

변경PR)
- 섬네일 세팅값은 width와 height 중 작은 값에 적용된다.
- 기존 섬네일의 width와 height 모두 현재 섬네일 세팅값과 다른 경우, 섬네일 세팅이 변경되었다고 판단한다.
- 제공되는 섬네일 세팅값이 250/600이기 때문에 250x600 (or 600x250) 사이즈 섬네일은 위 조건에서 섬네일 변경여부를 체크할 수 없으나 극소수일 것으로 판단해 보수적으로 조건을 정했다.